### DISCUSSION TITLE
Node Reward

### DISCUSSION THREAD

**Discussion by Doge369420** [dogecoin/dogecoin#2619] [General]
Would it be possible to divert say 10% of the block reward towards a node reward?
It seems to me that such a small incentive would greatly help with network and adoption, if it could be possible. 
I think it would also help with bringing in new people to the world of dogecoin, and incentives are a fun way to go about it.

**Comment by harshil21** [comment]
Duplicate of #2574

**Reply by Doge369420** [reply]
Thanks for your response. I have checked out the discussion you tagged and while being a similar topic about a node reward, what they proposed was an additional tax on transactions to help fund the doge foundation. What I'm proposing is diverting say 10% of the block reward from miners towards nodes. Since they play more than a 10% role in maintaining the network, I would say it's more than valid and would not be an 'extra transaction tax'. I believe the added benefits of more node adoption and fun novelty for newcomers would be a great feature.
